azure_functions_worker/testutils.py

```diff
@@ -33,6 +34,10 @@
 import requests
 
 from azure_functions_worker._thirdparty import aio_compat
+from azure_functions_worker.shared_memory_data_transfer.file_accessor_factory \
```

Comment:
I would ignore pytlint for the imports and let the lines be longer tbh

Reply:
Builds fail...

Reply:
😥